featuretools/tests/demo_tests/test_demo_data.py

```diff
@@ -1,29 +1,42 @@
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2
+
+
+import pytest
+
 from featuretools.demo import load_flight, load_mock_customer, load_retail
 from featuretools.synthesis import dfs
 
 
-def test_load_retail_diff():
+@pytest.fixture(autouse=True)
```

Comment:
if this fixture is autouse do we need to

Reply:
Fixed
https://github.com/Featuretools/featuretools/pull/542/commits/1cdad1be41a7fa99988c36bacc14c43fb8e856c7